app/models/review.rb

```diff
@@ -1,15 +1,16 @@
+# review
 class Review < ApplicationRecord
-
   # user will review many movies
-	belongs_to :user
-	# post will have many reviews
-	belongs_to :movie
+	belongs_to :user, class_name: 'User'
+  # movie will have many reviews
+	belongs_to :movie, class_name: 'Movie'
```

Comment:
@sdsunjay I don't believe class_name is necessary here. if the relationship is the same as the class name, no need to specify.

Reply:
https://github.com/sdsunjay/movie-recommendations-with-ml/pull/1#discussion_r221807622
I have removed the class names.